client/src/pages/spectrogram/components/PluginsPane.tsx

```diff
@@ -23,6 +24,9 @@ export interface PluginsPaneProps {
   setMeta: (meta: SigMFMetadata) => void;
 }
 
+const IQ_SIGMF_32 = 'iq/cf32_le';
```

Comment:
Wouldn't be better to move this to an enum  like we did with FeatureFlag?

Reply:
Done